docs/chart-and-series-types/bubble-series.md

```diff
@@ -61,6 +61,45 @@ In the following example, a color axis is added and the `x` value is used for co
 
 <iframe style="width: 100%; height: 480px; border: none;" src="https://www.highcharts.com/samples/embed/highcharts/coloraxis/changed-default-color-key" allow="fullscreen"></iframe>
 
+Blend colors
+----------
+It is also possible to show another dimention - density. By using the [bubble.blendColors](https://api.highcharts.com/highcharts/series.bubble.blendColors) option, you can add a bubble with multiple graphics (colors) where each color graphic has the same zIndex as all other graphics with the same colors. This allows to show a nice blend effect that could be very useful for showing density ([espetially in a map chart](https://www.highcharts.com/docs/maps/mapbubble-series#mapbubble-blend-colors)).
```

Comment:
A little spelling error: dimention -> dimension

Reply:
I just fixed this and a few other minor things, will push soon.